src/acl/external/eDirectory_userip/required.m4

```diff
@@ -5,4 +5,6 @@
 ## Please see the COPYING and CONTRIBUTORS files for details.
 ##
 
-AS_IF([test "x$LIBLDAP_LIBS" != "x"],[BUILD_HELPER="eDirectory_userip"])
+AS_IF([test "x$LIBLDAP_LIBS" != "x" -a "x$squid_host_os" != "xmingw"],[
+  BUILD_HELPER="eDirectory_userip"
```

Comment:
> ext_edirectory_userip_acl uses API that are too different from what is provided by windlap.h

ext_edirectory_userip_acl sources do not include windlap.h, so I assume they are using "API" from some other header file (perhaps ldap.h?). Is it possible to disable ext_edirectory_userip_acl build when that "other header file" is not available instead?

Reply:
Not easily.  AFAIK those interested in this helper do not include Windows people anyway so this is good enough.